test/e2e/components/vl-pager.js

```diff
@@ -1,10 +1,121 @@
 const { VlElement } = require('vl-ui-core');
+const { By } = require('selenium-webdriver');
 
 class VlPager extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+
+    async _getPagerList() {
+        return this.shadowRoot.findElement(By.css('#pager-list'));
+    }
+
+    async _getBounds() {
+        const pagerList = await this._getPagerList();
+        return pagerList.findElement(By.css('#bounds'));
+    }
+
+    async _getPages() {
+        const pagerList = await this._getPagerList();
+        return pagerList.findElement(By.css('#pages'));
+    }
+
+    async _pageNumberIsDisplayed(number) {
+        const pages = await this._getPages();
+        const pageNumberElements = await pages.findElements(By.css('li'));
+        const displayedNumbers = await Promise.all(pageNumberElements.map(e => e.getText()))
+        return displayedNumbers.includes(number);
+    }
+
+    async _navigateUntilPagenumberIsClickable(pageNumber) {
+        const boolean = await this._pageNumberIsDisplayed(pageNumber);
+        if(boolean) {
```

Comment:
Best nog de boolean hernoemen naar iets duidelijks (of het gewoon inlinen in de if)

Reply:
Fixed